src/uct/sm/scopy/base/scopy_ep.h

```diff
@@ -44,9 +44,6 @@ typedef ucs_status_t
                           uint64_t remote_addr, uct_rkey_t rkey,
                           uct_scopy_tx_op_t tx_op);
 
-typedef struct uct_scopy_tx_arb_elem {
```

Comment:
unrelated?

Reply:
related: this file is included into C++ gtest file & produces error